### ISSUE TITLE
Feature request: list view of child notes

### ISSUE THREAD

**Issue by msbentley** [zadam/trilium#548]
I'm currently trying to move away from a combination of traditional note-taking software (Evernote) and a hierarchical outliner (checkvist) and would prefer to end up with one tool (open source and self-hosted).

So far I've been looking at Joplin and Boostnote on the note side, but then came across Trilium. I think it has the potential to fit both of my use cases, but I would love a more structured view of child notes (i.e. a list view of titles and creation/modification times, preferably with sorting).

Essentially an option to switch the "block" display of child notes to a table/list view would be amazing.

Thanks! Mark

**Comment by zadam** [comment]
So if I understand it, you'd like something similar to 3-pane design of e.g. Evernote or Outlook - where we currently have the tree pane, detail pane, but not the note-list pane.

Frankly, I'm not really convinced this would be that useful. My approach is to integrate all the navigation/manipulation of notes into the tree pane instead.

**Comment by msbentley** [comment]
I guess I wouldn't need a third panel _per se_ but a way to order the child notes e.g. by date created, or date updated. The other change that would be useful for my workflow, would be to add the option to view the child notes as a list. i.e. instead of the blocks containing the title now:

![image](https://user-images.githubusercontent.com/1671065/58752783-194e0000-84b5-11e9-826f-383b49d5180a.png)

to have something like:

![Trilium Notes - notes_004](https://user-images.githubusercontent.com/1671065/58752877-8f069b80-84b6-11e9-95c6-0d35f90d3a59.png)

(excuse the horrible copy'n'paste mashup!).

I am of course open to changing my workflow, and haven't yet explored all that Trilium has to offer. Typically in Evernote I "live" at the top of a list of notes ordered by recently changed, to pick up where I left off. Perhaps there is already a way to do this?

**Comment by pcause** [comment]
agree.  like to see this view.  also, if the not has no content and children can the top of the display be smaller  so we can see moreof the children.  when there is content you can switch the css applied to be one that shows more of the note

**Comment by Sytten** [comment]
I was looking for something similar, I think a a type similar to Book but only displaying the title of the the children nodes would be very clean.

**Comment by meichthys** [comment]
@msbentley I know it's not exactly what you're looking for, but if you create a new tab and click the following button it will list the most recent pages you've edited:
![image](https://user-images.githubusercontent.com/10717998/83187118-a4c79480-a0fb-11ea-9bb0-66f759187a46.png)

**Comment by msbentley** [comment]
Thanks @meichthys I'll give it a try.

**Comment by msbentley** [comment]
I also wondered if this could be implemented with using the API/scripting? I know very little js, but if it's in principle possible it could be an interesting learning experience...

**Comment by zadam** [comment]
Yes, with a custom widget. But this is not really documented so I can't recommend doing that ATM.

**Comment by msbentley** [comment]
Hi @zadam I saw in the last few months a bunch of commits referencing a list view, but I guess that didn't make it to the 0.46  beta? Is this anything like the above, or something different? Many thanks!

**Comment by zadam** [comment]
You can now switch the "children overview" into a list view:

![](https://user-images.githubusercontent.com/617641/109065236-94286700-76eb-11eb-8840-7d1bf20892d1.png)

**Comment by msbentley** [comment]
Ahhhh, great, thanks @zadam! I was looking in the note type drop-down and didn't see the extra icons! Great!

I'll leave this ticket open, if it's OK, because I don't see an option to sort (e.g. by created/updated time), which for me would be the final "icing on the cake" :)
